### PR TITLE
Feat: add VaultItemUi component

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

File: src/components/v2/VaultItemUi/styles.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      justify-content: space-between;
+      margin-bottom: ${theme.spacing(6)};
+    `,
+    title: css`
+      display: flex;
+      align-items: center;
+    `,
+    tokenIcon: css`
+      width: ${theme.shape.iconSize.large};
+      height: ${theme.shape.iconSize.large};
```
Comment (therealemjy) [inline src/components/v2/VaultItemUi/styles.ts]: Check the theme and the warnings in the console: you need to add the units

File: src/components/v2/VaultItemUi/styles.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      margin-right: ${theme.spacing(1)};
+    `,
+    tokenIconLarge: css`
+      width: ${theme.shape.iconSize.xLarge};
+      height: ${theme.shape.iconSize.xLarge};
```
Comment (therealemjy) [inline src/components/v2/VaultItemUi/styles.ts]: Check the theme and the warnings in the console: you need to add the units

File: src/components/v2/VaultItemUi/index.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  userStakedWei,
+  stakingAprPercentage,
+  dailyEmissionWei,
+  totalStakedWei,
+  onClaim,
+  onStake,
+  onReward,
+  className,
+}: IVaultItemUiProps) => {
+  const classes = useStyles();
```
Comment (therealemjy) [inline src/components/v2/VaultItemUi/index.tsx]: To be consistent with the rest of the codebase:
```suggestion
  const styles = useStyles();
```

File: src/components/v2/VaultItemUi/index.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+export const VaultItemUiDefault = () => (
+  <VaultItemUi
+    tokenId="vai"
+    rewardTokenId="xvs"
+    rewardWei={new BigNumber('000900000000000000')}
+    userStakedWei={new BigNumber('100000000000000000000')}
+    stakingAprPercentage={2.39}
+    dailyEmissionWei={new BigNumber('2120000000000000000')}
+    totalStakedWei={new BigNumber('1233000000000000000000')}
+    onClaim={console.log}
```
Comment (coreyar) [inline src/components/v2/VaultItemUi/index.stories.tsx]: To reduce lint warnings lets use a noop function here instead of console.log

**Review by therealemjy — APPROVED** [review]

**Pull request merged**